nix/nim-status/getFlags.nix

```diff
@@ -0,0 +1,119 @@
+{ stdenv, lib
+# Dependencies
+, xcodeWrapper
+, androidPkgs
+, platform ? "android"
+, arch ? "386"
+, fromNim ? false
+, api ? "23" } :
+
+
+let
+  ANDROID_NDK_HOME = "${androidPkgs}/ndk-bundle";
+
+
+  iosArchMap = {
+    "386" = "x86_64";
+    "arm" = "armv7";
+    "arm64" = "arm64";
+  };
+  iosArch = lib.getAttr arch iosArchMap;
```

Comment:
Stuff like `iosArchMap` or `targetArchMap` could be made less verbose by just combining it with the `getAttr` call:
```nix
{
  iosArch = lib.getAttr arch {
    "386" = "x86_64";
    "arm" = "armv7";
    "arm64" = "arm64";
  };
}
```
Also, some comments explaining the purpose of these maps would helps someone reading this.

Reply:
Done.